scio-core/src/main/scala/com/spotify/scio/ScioContext.scala

```diff
@@ -103,16 +111,19 @@ object ScioContext {
  * @groupname Ungrouped Other Members
  */
 // scalastyle:off number.of.methods
-class ScioContext private[scio] (val options: DataflowPipelineOptions, testId: Option[String]) {
+class ScioContext private[scio] (val options: DataflowPipelineOptions, private var artifacts: List[String], testId: Option[String]) {
 
   import Implicits._
 
+  private val LOG = LoggerFactory.getLogger(ScioContext.getClass)
```

Comment:
Minor: it's called `logger` everywhere else in the scala code base. Better be consistent but what should we use?